Source/WebCore/rendering/LayoutRepainter.h

```diff
@@ -32,9 +32,11 @@ namespace WebCore {
 class RenderElement;
 class RenderLayerModelObject;
 
+enum class RepaintOutlineBox : bool { No, Yes };
```

Comment:
Can we call this `RepaintOutlineBounds` for consistency with `outlineBoundsForRepaint()`?